plugins/protocolbuffers/java/v23.3/buf.plugin.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/protocolbuffers/java
+plugin_version: v23.3
+source_url: https://github.com/protocolbuffers/protobuf
+description: Base types for Java. Generates message and enum types.
+output_languages:
+  - java
+spdx_license_id: BSD-3-Clause
+license_url: https://github.com/protocolbuffers/protobuf/blob/v23.3/LICENSE
+registry:
+  maven:
+    deps:
+      - com.google.protobuf:protobuf-java:3.23.3
+    additional_runtimes:
+      - name: lite
+        deps:
+          - com.google.protobuf:protobuf-javalite:3.23.3
+          - build.buf:protobuf-javalite:3.23.3
```

Comment:
We used to conditionally add this if needed on the BSR side of things, but this feels more maintainable long term to add it to the plugin's dependencies.